sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java

```diff
@@ -115,7 +106,7 @@ public void registerViews(Navigator navigator) {
 	}
 
 	public void create() {
-		create(null);
+		create((CaseReferenceDto) null);
```

Comment:
Because I overloaded the create method with `create(EventParticipantReferenceDto eventParticipantRef)` and now he can no longer know which version of `create` with 1 parameter  to choose if this parameter isn't typed

I did this so I wouldn't have to change the current behavior. But I can also refactor it to add a proper dedicated create method without any parameter.

What do you prefer ?

Reply:
No, that's fine then!